source/examples/rapids-coiled-cudf/notebook.ipynb

```diff
@@ -0,0 +1,2461 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %load_ext cudf.pandas
</pre><p>We might want to add a note in the text above, about this is needed to be imported at the top, <a href="https://docs.rapids.ai/api/cudf/stable/cudf_pandas/usage#jupyter-notebook-or-ipython-usage" target="_blank">https://docs.rapids.ai/api/cudf/stable/cudf_pandas/usage/#jupyter-notebook-or-ipython-usage</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>